classic/src/view/panel/layer/Metadata.js

```diff
@@ -12,317 +12,485 @@ Ext.define('MoMo.admin.view.panel.layer.Metadata',{
+                        fieldLabel: '{i18n.metadata.referenceDate}',
+                        value: '{metadata.referenceDate}'
+                    }
+                }, {
+                    xtype: 'textfield',
+                    name: 'metadata-format',
+                    bind: {
+                        fieldLabel: '{i18n.metadata.format}',
+                        value: '{metadata.format}'
+                    }
+                }, {
+                    xtype: 'textfield',
+                    name: 'metadata-limitations',
+                    bind: {
+                        fieldLabel: '{i18n.metadata.limitations}',
+                        value: '{metadata.limitations}'
+                    }
+                }, {
+                    xtype: 'textfield',
+                    name: 'metadata-onlineResource',
+                    bind: {
+                        fieldLabel: '{i18n.metadata.onlineResource}',
+                        value: '{metadata.onlineResource}'
                     }
                 }, {
                     xtype: 'textfield',
-                    name: 'metadata-organisation-website',
+                    name: 'metadata-dataSource',
+                    disabled: true,
                     bind: {
-                        fieldLabel: '{i18n.metadata.website}',
-                        value: '{metadata.organisation.website}'
+                        fieldLabel: '{i18n.metadata.dataSource}',
+                        value: '{metadata.dataSource}'
                     }
                 }, {
+                    xtype: 'textfield',
+                    name: 'metadata-publications',
+                    disabled: true,
+                    bind: {
+                        fieldLabel: '{i18n.metadata.publications}',
+                        value: '{metadata.publications}'
+                    }
+                }]
+            }, {
+                xtype: 'fieldcontainer',
+                flex: 1,
+                defaults: {
+                    width: '100%',
+                    labelWidth: 150
+                },
+                items: [{
                     xtype: 'fieldset',
-                    name: 'metadata-organisation-address',
+                    name: 'metadata-organisation',
                     width: '100%',
                     bind: {
-                        title: '{i18n.metadata.address}'
+                        title: '{i18n.metadata.organisation}'
                     },
                     defaults: {
                         width: '100%'
                     },
                     items: [{
                         xtype: 'textfield',
-                        name: 'metadata-organisation-address-deliveryPoint',
+                        name: 'metadata-organisation-name',
                         allowBlank: false,
+                        msgTarget: 'under',
+                        validator: function(val) {
+                            var startsWith = val.startsWith(' ');
```

Comment:
The validator function has been moved in a new util class MoMo.admin.util.TextfieldValidator. Here Ext.String.startsWith() ist used. See 96e4936